cmd/cli/evaluate.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/thomaspoignant/go-feature-flag/ffuser"
+	"github.com/thomaspoignant/go-feature-flag/retriever"
+	"github.com/thomaspoignant/go-feature-flag/variation"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+type EvaluationResult struct {
+	TrackEvents   bool        `json:"trackEvents"`
+	VariationType string      `json:"variationType"`
+	Failed        bool        `json:"failed"`
+	Version       string      `json:"version"`
+	Reason        string      `json:"reason"`
+	ErrorCode     string      `json:"errorCode"`
+	Value         interface{} `json:"value"`
+	Cacheable     bool       `json:"cacheable"`
+}
```

Comment:
**question:** Why do you need to create a new response struct here? You can probably use `model.VariationResult` instead that has the exact same signature _(except that it is a generic type)_.

This struct is available [here](https://github.com/thomaspoignant/go-feature-flag/blob/8f53d070de7ba82a626275348b1f57f2e353f2f5/model/variation_result.go#L11).